.changelog/unreleased/breaking-changes/1323-encode-vec-no-error.md

```diff
@@ -0,0 +1,4 @@
+- Align the return signature of the `encode_vec` and
```

Comment:
Should we have the component name somewhere, either here or in the directory structure under `.changelog`? Don't remember what the convention was.

Reply:
[67a28a3](https://github.com/informalsystems/tendermint-rs/pull/1324/commits/67a28a31fec3ce962b73a03de9895a16f64451fb)